apps/server/src/utilities/pathUtil.ts

```diff
@@ -1,27 +1,21 @@
 import getPath from 'platform-folders';
 import * as path from 'path';
 import { Logger } from '@nestjs/common';
-import { homedir } from 'os';
-import { execFileSync } from 'child_process';
 
 export const getSimbridgeDir = () => {
   try {
     return path.join(getPath('documents'), 'FlyByWireSim', 'Simbridge');
   } catch (e) {
-    Logger.warn('Could not get documents path via WinAPI, trying alternate method', e);
+    Logger.warn(
+      'Could not get documents path via WinAPI, Windows Controlled Folder Access is likely blocking the app. Using AppData as fallback',
+      e,
+    );
   }
   try {
-    const output = execFileSync('Powershell.exe', ['-Command', `[System.Environment]::GetFolderPath('MyDocuments')`]);
-    const documents = output.toString().trim();
-    if (!documents) {
-      throw new Error('Path is empty');
-    }
-    return path.join(documents, 'FlyByWireSim', 'Simbridge');
+    return path.join(getPath('appdata'), 'FlyByWireSim', 'Simbridge');
   } catch (e) {
-    Logger.warn('Could not get documents path via Powershell, trying to use %USERPROFILE% method', e);
+    Logger.warn('Could not get AppData path via WinAPI. Giving up.', e);
```

Comment:
I would recommend escalating this to an error rather than a warning.

Reply:
Changed 👍